boards/lilygo/ttgo_lora32/doc/index.rst

```diff
@@ -205,7 +205,7 @@ message in the monitor:
    ***** Booting Zephyr OS vx.x.x-xxx-gxxxxxxxxxxxx *****
    Hello World! ttgo_lora32
 
-LoRa samples
+Code samples
```

Comment:
@kartben Sounds this file was not updated to latest doc design. Should we do it before v4.0.0?

Reply:
sure, we can try :) #81411

Reply:
It didn't build out of the box because the device tree lacked "disk-name = "SD";".
I added it, it now builds, and I tested it with the "fs_sample".
Also rebased on to v4.0.0